test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -375,8 +375,7 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 			}
 		})
 		It("[test_id: 59572] Check RPS Mask is applied to at least one single rx queue on all veth interface", func() {
-			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
-				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+			if !profileutil.IsRpsEnabled(profile) {
```

Comment:
isRpsEnabled checks if Profile is annotations related to RPS,   By default in our E2E testing we don't add any RPS Annotations